packages/connect-node-conformance/conformance.yaml

```diff
@@ -0,0 +1,21 @@
+features:
```

Comment:
Minor nits that can help clean this up:

* You can omit `protocols` and `codecs` since all options are supported (if absent, all are assumed)
* You can also omit `supportsTls`, `supportsConnectGet`, `requiresConnectVersionHeader`, and `supportsMessageReceiveLimit` since you have them listed as the defaults and if absent, the defaults are assumed.

Reply:
I did think of omitting the defaults but it just felt easier to parse and see what is included just by looking at the config. I'll omit them in a separate PR.